xline/src/server/kv_server.rs

```diff
@@ -433,7 +433,7 @@ impl Kv for KvServer {
     ) -> Result<tonic::Response<TxnResponse>, tonic::Status> {
         debug!("Receive TxnRequest {:?}", request);
         Self::check_txn_request(request.get_ref())?;
-        let is_fast_path = true;
+        let is_fast_path = false; // lock need revision of txn
         let (cmd_res, sync_res) = self.propose(request, is_fast_path).await?;
 
```

Comment:
If we use slow path here, should `sync_res` always be Some(...)? Then we should at least give a warning if it's None. I think a better way is to seperate self.propose into two functions. E.g., propose_slow will always `SyncResponse`, while propose_fast will return `Option<SyncResponse>`

Reply:
In the future we may let the client decide whether to use fast path，so I think current logic is better for this.